test/Libraries/WorkflowTests/PackageValidationTest.cs

```diff
@@ -89,7 +89,7 @@ public void TestTextFromDynamoText()
         public void ConvertMeshIntoLinePointsSurfaces()
         {
             var testFilePath = Path.Combine(TestDirectory,
-    @"core\userdata\0.8\packages\MeshToolkit.0.8.2\extra\ConvertMeshIntoLinePointsSurfaces.dyn");
+    @"core\userdata\0.8\packages\MeshToolkit\extra\ConvertMeshIntoLinePointsSurfaces.dyn");
```

Comment:
@mmisol Did we also update the MeshToolKit version? If yes, then I prefer not putting it in `0.8` folder anymore but reflect the actual version here

Reply:
Ok, I could do that. Should I do something to the other packages in that folder? We have:
- List.SortByKeyList version 0.0.2
- Dynamo Text version 0.1.0

Reply:
Thinking about it, maybe `0.8` refers to the version of Dynamo? Would that explain why the `backup` folder got created?

Reply:
I would leave them there, I think we have another task for Dynamo Text package.